src/main/java/org/broadinstitute/hellbender/engine/spark/SparkCommandLineArgumentCollection.java

```diff
@@ -45,4 +59,24 @@ public String getSparkMaster() {
         return sparkMaster;
     }
 
+    public String getSparkVerbosity(final Log.LogLevel toolVerbosity) {
+        if (sparkVerbosity != null) return sparkVerbosity;
+        if (toolVerbosity.equals(Log.LogLevel.DEBUG)) {
```

Comment:
A switch statement might be somewhat cleaner here.

Reply:
Using a switch would be clearer but might be complicated since these are objects